### PR TITLE
https-dns-proxy: Version bump

### PR DESCRIPTION
**Pull request by aarond10**
Now includes support for SRV records, cap on concurrent requests.

Signed-off-by: Aaron Drew <aarond10@gmail.com>

### CODE REVIEW

File: net/https-dns-proxy/Makefile [hunk 1]
```diff
@@ -1,14 +1,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=https_dns_proxy
-PKG_VERSION:=2016-09-04
+PKG_VERSION:=2017-01-07
 PKG_RELEASE=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
```
Comment (diizzyy) [inline net/https-dns-proxy/Makefile]: Please use xz here instead of bz2

**Comment by ryzhovau** [comment]
As I see [here](https://github.com/openwrt/packages/blob/master/net/https-dns-proxy/files/https_dns_proxy.init#L19)
```
	procd_set_param command ${PROG} \
		-l "$listen_addr" -p "$listen_port" \
		-u "$user" -g "$group"
```
while `-l` option is a path to log file. It should be replaced to `-a`.

**Comment by aarond10** [comment]
Ouch. That argument issue was embarrassing...

I've been using it for about a year, but I'm the author... 
Is there something special about your config that might stop it working? 
Happy to try to help.

**Comment by ryzhovau** [comment]
Thanks for responding, I'll send you a mail as long as it's off-topic here.

**Pull request merged**